### PR TITLE
chore(website): remove config auth mode doc

### PR DESCRIPTION
**Pull request by SunnyBoy-WYH**

### CODE REVIEW

File: content/cn/docs/config/config-authentication.md [hunk 1]
```diff
@@ -64,35 +62,7 @@ auth.graph_store=hugegraph
...
-auth.authenticator=org.apache.hugegraph.auth.ConfigAuthenticator
-auth.admin_token=token-value-a
-auth.user_tokens=[hugegraph1:token-value-1, hugegraph2:token-value-2]
-```
-
-在配置文件`hugegraph{n}.properties`中配置`gremlin.graph`信息：
-
-```properties
-gremlin.graph=org.apache.hugegraph.auth.HugeFactoryAuthProxy
-```
```
Comment (imbajin) [inline content/cn/docs/config/config-authentication.md:95]: shall we mark them as `outdated` & put them in a single space? (or just delete them? @javeme )

Comment (SunnyBoy-WYH) [reply content/cn/docs/config/config-authentication.md:95]: > shall we mark them as `outdated` & put them in a single space? (or just delete them? @javeme )

i think we just delete it; In the future the best method is do whole doc's version control overall，not record the specific change;

and another way , we can record  `doc change log`, like current Hugegraph `change log`

Comment (SunnyBoy-WYH) [reply content/cn/docs/config/config-authentication.md:95]: > > shall we mark them as `outdated` & put them in a single space? (or just delete them? @javeme )
> 
> i think we just delete it; In the future the best method is do whole doc's version control overall，not record the specific change;
> 
> and another way , we can record `doc change log`, like current Hugegraph `change log`

cc @javeme

Comment (javeme) [reply content/cn/docs/config/config-authentication.md:95]: Is the ConfigAuthenticator still available? I think we can delete this doc if NO or mark outdated if YES.
The version control of documents is the direction we need to move towards.

Comment (imbajin) [reply content/cn/docs/config/config-authentication.md:95]: > > shall we mark them as `outdated` & put them in a single space? (or just delete them? @javeme )
> 
> i think we just delete it; In the future the best method is do whole doc's version control overall，not mark the specific change;
> 
> and another way , we can record  `doc change log`, like current Hugegraph Release `change log`
> 
> cc @javeme 

OK，just delete it for now

outdated doc should be visited by website tag/version function，record this as a important need（issue）

**Review by imbajin — APPROVED** [review]

File: content/cn/docs/images/images-server/621swaggerui设置鉴权位置示例.jpg
Comment (imbajin) [inline content/cn/docs/images/images-server/621swaggerui设置鉴权位置示例.jpg:1]: better not use CN chars in filename~ u could enhance it in future PRs ~

**Review by imbajin — APPROVED** [review]
THX

**Pull request merged**